writing-specifications.rst

```diff
@@ -1,40 +1,45 @@
-Writing Specifications
-======================
+Write a Gauge specification
+===========================
 
-In this page, we'll cover everything you need to know about writing Gauge specification. Before we start, do take a look at the IDE plugins for Gauge that improve your test authoring experience.  
+This page provides information about what a specification is, how to write a specification, and other related information such as concepts, step implementation, specs directory, and so on.
+
+Specifications (specs)
+----------------------
+
+A specification is a business test case which describes a particular feature of the application that needs testing.  
+Gauge specifications support a ``.spec`` or ``.md`` file format and these specifications are written in a syntax similar to Markdown.
+
+For more information about Markdown, see `Markdown Syntax <https://daringfireball.net/projects/markdown/syntax>`__.
+
+Text editor for specs
+^^^^^^^^^^^^^^^^^^^^^
+
+You can author Gauge specifications by using any text editor. 
+It is recommended that you use one of the following IDE plugins:
 
 .. tab-container:: ide
 
     .. tab:: Visual Studio Code
 
-        Gauge projects can be created and executed in Visual Studio Code using the `Gauge extension for VSCode <https://marketplace.visualstudio.com/items?itemName=getgauge.gauge>`__.
-        This plugin currently supports Gauge with JavaScript, Ruby and Python.
+        Gauge projects can be created and run in Visual Studio Code by using the `Gauge extension for VSCode <https://marketplace.visualstudio.com/items?itemName=getgauge.gauge>`__.
+        This plugin currently supports Gauge with JavaScript, Ruby, and Python.
 
     .. tab:: IntelliJ Idea
 
-        Gauge projects can be created and executed in IntelliJ IDEA using the `Gauge plugin for IntelliJ IDEA <https://github.com/getgauge/Intellij-Plugin/blob/master/README.md>`__.
+        Gauge projects can be created and run in IntelliJ IDEA by using the `Gauge plugin for IntelliJ IDEA <https://github.com/getgauge/Intellij-Plugin/blob/master/README.md>`__.
         This plugin only supports Gauge with Java.
 
     .. tab:: Visual Studio
 
-        Gauge projects can be created and executed in Visual Studio using the `Gauge extension for Visual Studio <https://github.com/getgauge/gauge-visualstudio/blob/master/README.md>`__.
+        Gauge projects can be created and run in Visual Studio by using the `Gauge extension for Visual Studio <https://github.com/getgauge/gauge-visualstudio/blob/master/README.md>`__.
         This plugin currently supports Gauge with C#.
```

Comment:
Modify this line to

This plugin currently supports Gauge with C# (.Net framework).